packages/account/src/Components/forms/confirmation-checkbox/confirmation-checkbox.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Checkbox, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { useFormikContext } from 'formik';
+
+/**
+ * Props for the confirmation checkbox component.
+ */
+type TConfirmationCheckboxProps = {
+    /**
+     * The label of the checkbox.
+     */
+    label: React.ReactNode;
+    /**
+     * The size of the checkbox label.
+     */
+    label_size?: 'xxxxs' | 'xxxs' | 'xxs' | 'xs' | 's' | 'sm' | 'm' | 'l' | 'xl' | 'xxl';
```

Comment:
Do we need to type this? by right it should follow Text component types.

Reply:
These are the possible `size` prop values for `Text` component based on our theme config, yet `Text` component accepts `string`